### PR TITLE
MultiViewport Transitions: Add example, and fix `isMapSynced`.

### PR DESCRIPTION
**Pull request by 1chandu**
- Create multi-viewport transition example using existing first-person-map example.
- Add bias to 'isMapSynched' to avoid flashes during transition.

Verified with layer-browser.

### CODE REVIEW

File: examples/wip/viewport-animation-first-person-map/app.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
+/* global window, document, fetch */
+import React, {Component} from 'react';
```
Comment (ibgreen) [inline examples/wip/viewport-animation-first-person-map/app.js]: Nit: Do we need two examples? Can't we just have one?

File: src/core/viewports/web-mercator-viewport.js [hunk 2]
```diff
@@ -267,14 +267,16 @@ export default class WebMercatorViewport extends Viewport {
...
+    const EPSILON = 0.000001;
     const MAPBOX_LIMITS = {
       pitch: 60,
       zoom: 40
     };
 
     const {pitch, zoom} = this;
 
-    return pitch <= MAPBOX_LIMITS.pitch && zoom <= MAPBOX_LIMITS.zoom;
+    return pitch <= (MAPBOX_LIMITS.pitch + EPSILON) &&
```
Comment (ibgreen) [inline src/core/viewports/web-mercator-viewport.js]: Normally one does something like `Math.abs(a - b) < EPSILON` to make sure signs etc don't affect the results. As one option. math.gl exports an `equals` function that does this.

File: examples/wip/viewport-animation-first-person-map/app.js (hunk 1 above)
Comment (1chandu) [reply examples/wip/viewport-animation-first-person-map/app.js]: I will merge them into one in future PR.

File: src/core/viewports/web-mercator-viewport.js (hunk 2 above)
Comment (1chandu) [reply src/core/viewports/web-mercator-viewport.js]: We probably should add lessOrEqual, greatOrEqual version to math as well.

**Pull request merged**